log/logger_test.go

```diff
@@ -0,0 +1,162 @@
+package log
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/zimmski/osutil"
+	"github.com/zimmski/osutil/bytesutil"
+)
+
+func TestLoggerWith(t *testing.T) {
```

Comment:
awesoooome

tiny nit: just for fun, I'd log something to the final logger objects you create... cause you check now that the files are created but just to make sure stuff ends up there